src/connectedk8s/azext_connectedk8s/_constants.py

```diff
@@ -206,13 +207,15 @@
 Cluster_Diagnostic_Checks_Job_Not_Scheduled = 'Unable to schedule cluster-diagnostic-checks job'
 Cluster_Diagnostic_Checks_Job_Not_Complete = 'Unable to complete cluster-diagnostic-checks job after scheduling'
 Pre_Onboarding_Diagnostic_Checks_Execution_Failed = 'Exception occured while trying to execute pre-onboarding diagnostic checks'
+Outbound_Connectivity_Check_Failed = "Outbound network connectivity check failed"
 Outbound_Connectivity_Check_Failed_For_Onboarding = "Outbound network connectivity check failed for onboarding"
 DNS_Check_Failed = "DNS Resolution failed"
 Cluster_Diagnostic_Prechecks_Failed = "Cluster diagnostic prechecks failed"
 Cluster_Diagnostic_Prechecks_Incomplete = "Cluster diagnostic prechecks failed to complete"
 Cluster_Diagnostic_Checks_Pod_Description_Save_Failed = 'Failed to save cluster diagnostic checks pod description'
 Cluster_Diagnostic_Checks_Job_Log_Save_Failed = 'Failed to save cluster diagnostic checks job log'
 # Diagnostic Results Name
+Outbound_Connectivity_Check_Result_String = "Outbound Network Connectivity Result:"
 Outbound_Connectivity_Check_Check_String = "Outbound Network Connectivity Check"
```

Comment:
Can't we combine 218 and 219 and make it just "Outbound Network Connectivity" ? Since we use contains at both places, this single string should work right?